qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -69,35 +70,74 @@ def __init__(
         self._properties = None
         self._defaults = None
 
-    def run(self, qobj: Qobj, job_name: Optional[str] = None) -> IBMQJob:
+    def run(
+            self,
+            qobj: Qobj,
+            job_name: Optional[str] = None,
+            job_share_level: Optional[str] = None
+    ) -> IBMQJob:
         """Run a Qobj asynchronously.
 
         Args:
             qobj: description of job.
             job_name: custom name to be assigned to the job. This job
                 name can subsequently be used as a filter in the
                 ``jobs()`` function call. Job names do not need to be unique.
+            job_share_level: allows sharing a job at the hub/group/project and
+                global level. The possible job share levels are: "global", "hub",
+                "group", "project", and "none".
+                    * global: the job is public to any user.
+                    * hub: the job is shared between the users in the same hub.
+                    * group: the job is shared between the users in the same group.
+                    * project: the job is shared between the users in the same project.
+                    * none: the job is not shared at any level.
+                If the job share level is not specified, then the job is not shared at any level.
+
+                Note:
```

Comment:
Can you remove the note and the example, mostly for brevity? It is useful information, but a bit more suited to the upcoming release notes or README tweaks and to the IQX documentation.